packages/dialtone-vue2/recipes/cards/ivr_node/ivr_node.mdx

```diff
@@ -250,7 +250,7 @@ A node in IVR like branch that has a condition instead of dtmf connector.
   :node-type="branch"
 >
   <template #connector>
-    <div class="ivr-connector d-px8 d-h24 d-bar-pill d-mbn12 d-fc-neutral-white d-fs-100">Add branch</div>
+    <div class="dt-recipe-ivr-node__connector d-px8 d-h24 d-bar-pill d-mbn12 d-fc-neutral-white d-fs-100">Add branch</div>
```

Comment:
Should these CSS utilities values be folded into `.dt-recipe-ivr-node__connector`? There are a few others like that throughout this diff.

Reply:
yeah I think so for sure